test/test_gpu_fused_reduction.cpp

```diff
@@ -1279,10 +1279,10 @@ TEST_F(NVFuserTest, FusionGroupAllreduce5_CUDA) {
   fe.compileFusion(&fusion, aten_inputs);
   auto outputs = fe.runFusion(aten_inputs);
 
-  auto t3 = t0 / t0.sum({0}).unsqueeze(0).to(at::kComplexDouble);
```

Comment:
Is this a problem? I think I did this type cast to reduce rounding errors.

Reply:
I don't see any failure, but I just added it back just in case.

Reply:
Rounding errors could depend on devices. I tend to do this anytime reduction is involved.